src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -61,6 +61,9 @@ int xclGetDebugProfileDeviceInfo(xclDeviceHandle handle, xclDebugProfileDeviceIn
 #define XCL_INVALID_SENSOR_VAL 0
 
 #define indent(level)   std::string((level) * 4, ' ')
+
+#define HEX_02X std::hex << std::uppercase << std::setw(2) << std::setfill('0')
```

Comment:
this is probably not very c++-ish?

Reply:
Yes, it isn't. Please use boost::format, and since it is only used in one place we don't need to define it up here